Testing/SystemTests/tests/analysis/VesuvioTesting.py

```diff
@@ -0,0 +1,514 @@
+from mantid.simpleapi import *
```

Comment:
Maybe these could go in scripts/Inelastic/vesuvio together with other common useful functions? Probably @martyngigg has a better idea what's the best place for this.